### ISSUE TITLE
built in deno types paging

### ISSUE THREAD

**Issue by littledivy** [denoland/deno#13773]
`deno types` output can be huge and its hard to read anything without paging through `less` or `grep`ing specifics. We should implement a built-in pager like `man`.

https://github.com/arijit79/minus is a nice crate for that

**Comment by kitsonk** [comment]
Well, it isn't really readable as a man page.  It isn't like `deno doc`.  It is really only useful for a few folks who need it to pipe it to a file, so that seems redundant.  Is there another case for `deno types`?  Feels like paging the output of `deno doc --json`, why would you do that?  If someone really wants to page it, well as you mention, `less` or `grep` are suitable.

**Comment by bartlomieju** [comment]
I agree with @kitsonk. This is something that should be left to other programs.

**Comment by bartlomieju** [comment]
I'm gonna close this issue for now, as it doesn't have much traction. Feel free to reopen if you feel strongly about it Divy.
